crates/rpc/rpc/src/eth/error.rs

```diff
@@ -26,6 +26,9 @@ pub enum EthApiError {
     PoolError(RpcPoolError),
     #[error("Unknown block number")]
     UnknownBlockNumber,
+    /// Thrown when querying for `finalized` or `safe` before the merge transition is finalized
+    #[error("Unknown block")]
```

Comment:
I'd also like to add the execution api reference here

I'd like to rename this no `UnknownSafeOrFinalizedBlock` just to prevent any confusion because we also have `UnknownBlockNumber` already